libs/ui-lib/lib/cim/components/Agent/BMCForm.tsx

```diff
@@ -187,14 +187,14 @@ const emptyValues: AddBmcValues = {
   online: true,
   nmState:
     'interfaces:\n\
-- name: <nic1_name>\n\
-  type: ethernet\n\
-  state: up\n\
-  ipv4:\n\
-    address:\n\
-    - ip: <ip_address>\n\
-      prefix-length: 24\n\
-    enabled: true\n\
+  - name: <nic1_name>\n\
```

Comment:
It's safer to use multi-line template literal to preserve formatting. We can avoid explicit `\n` and be sure about white spaces at the beginning of a line.